src/main/java/dev/j3fftw/litexpansion/ItemSetup.java

```diff
@@ -145,9 +147,9 @@ private void registerEndgameItems() {
             null, Items.UU_MATTER, null,
             Items.UU_MATTER, Items.UU_MATTER, Items.UU_MATTER
         );
-        registerItem(Items.IRIDIUM_PLATE, RecipeType.ENHANCED_CRAFTING_TABLE,
+        registerItem(Items.IRIDIUM_PLATE, MetalForge.METAL_FORGE,
             Items.IRIDIUM, Items.ADVANCED_ALLOY, Items.IRIDIUM,
-            Items.ADVANCED_ALLOY, new ItemStack(Material.DIAMOND), Items.ADVANCED_ALLOY,
+            Items.ADVANCED_ALLOY, null, Items.ADVANCED_ALLOY,
```

Comment:
how come you removed the diamond from this?

Reply:
it removes the diamond block underneath so technically i added 8 diamonds

Reply:
Don't care. Want a diamond in recipe regardless

Reply:
oki, then there is an issue, i cheesed it. because when the inv is full it doesnt wanna produce the item